src/DssSpell.sol

```diff
@@ -45,69 +52,110 @@ contract DssSpellAction is DssAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
-    uint256 constant THREE_PCT = 1000000000937303470807876289;
-    uint256 constant FOUR_PCT  = 1000000001243680656318820312;
+    uint256 constant TWO_PCT            = 1000000000627937192491029810;
+    uint256 constant TWO_PT_FIVE_PCT    = 1000000000782997609082909351;
+    uint256 constant THREE_PCT          = 1000000000937303470807876289;
+    uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
+    uint256 constant FOUR_PCT           = 1000000001243680656318820312;
+    uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
+    uint256 constant FIVE_PT_FIVE_PCT   = 1000000001697766583380253701;
+    uint256 constant SIX_PCT            = 1000000001847694957439350562;
+    uint256 constant SEVEN_PT_FIVE_PCT  = 1000000002293273137447730714;
 
     /**
         @dev constructor (required)
-        @param lib         address of the DssExecLib contract
         @param officeHours true if officehours enabled
     */
-    constructor(address lib, bool officeHours) public DssAction(lib, officeHours) {}
-
-    uint256 constant MILLION = 10**6;
-
-    address constant UNIV2DAIUSDC_GEM   = 0xAE461cA67B15dc8dc81CE7615e0320dA1A9aB8D5;
-    address constant UNIV2DAIUSDC_JOIN  = 0xA81598667AC561986b70ae11bBE2dd5348ed4327;
-    address constant UNIV2DAIUSDC_FLIP  = 0x4a613f79a250D522DdB53904D87b8f442EA94496;
-    address constant UNIV2DAIUSDC_PIP   = 0x25CD858a00146961611b18441353603191f110A0;
-
-    address constant UNIV2ETHUSDT_GEM   = 0x0d4a11d5EEaaC28EC3F61d100daF4d40471f1852;
-    address constant UNIV2ETHUSDT_JOIN  = 0x4aAD139a88D2dd5e7410b408593208523a3a891d;
-    address constant UNIV2ETHUSDT_FLIP  = 0x118d5051e70F9EaF3B4a6a11F765185A2Ca0802E;
-    address constant UNIV2ETHUSDT_PIP   = 0x9b015AA3e4787dd0df8B43bF2FE6d90fa543E13B;
+    constructor(bool officeHours) public DssAction(officeHours) {}
+
+    uint256 constant WAD        = 10**18;
+    uint256 constant RAD        = 10**45;
+    uint256 constant MILLION    = 10**6;
+
+    bytes32 constant ETH_A_ILK          = "ETH-A";
```

Comment:
For the next spell I'd suggest we use directly the ilk names everywhere which are self descriptive and as complex as the variable. It seems a bit easier than adding extra lines and having to remember the new names.

Reply:
The problem with repeating strings is that if you typo one then you don't get a compiler error, but if you typo a variable name you will which is why I prefer not to inline the string names.